src/utils.py

```diff
@@ -16,6 +16,14 @@ def fps_from_filename(filename: str):
     else:
         return None
 
+def width_height_from_bitmap(bitmap):
+    # Detect sprite orientation
+    # if there is no size in the filename, each frame is considered to be a square
+    if (bitmap.height > bitmap.width):
+        return bitmap.width, bitmap.width
+    else:
+        return bitmap.height, bitmap.height
+
```

Comment:
It seems you reverted the change I made earlier. Did I re-introduce a bug?

Reply:
Unfortunately, if the bitmap filename does not contain the tile dimensions, the code was breaking.
dota-diretide.bmp is 32x256, without this code, the height will be set to 256.

Deleting only this function is also inconsistent with `frame_count_from_bitmap` that uses the same logic to count the number of tiles in the tilesheet.